browser/brave_vpn/sources.gni

```diff
@@ -15,7 +15,21 @@ if (enable_brave_vpn) {
     "//brave/browser/brave_vpn/vpn_utils.cc",
     "//brave/browser/brave_vpn/vpn_utils.h",
   ]
-
+  if (is_win) {
+    brave_browser_brave_vpn_sources += [
+      "//brave/browser/brave_vpn/dns/brave_vpn_dns_observer_factory.cc",
```

Comment:
why are all of these going in sources.gni?

Reply:
same as others, should I move it to a BUILD.gn?